G.A.M.M.A/modpack_addons/A Few Random Fixes/gamedata/scripts/faction_stocks.script

```diff
@@ -0,0 +1,7 @@
+function get_trader_community()
+  -- noop
+end
+
+function on_game_start()
+  UnregisterScriptCallback("ActorMenu_on_trade_started", a_faction_prices.ActorMenu_on_trade_started)
+end
```

Comment:
https://github.com/bellyillish/afrf/blob/main/README.md#f5

The issue is in the file `C:\Gamma\mods\G.A.M.M.A. UI\gamedata\scripts\a_faction_prices.script`. This file is in the current download version, but I don't see it here in the repo. Maybe you already removed it. If so disregard this file.